modules/Hoot/managers/api.js

```diff
@@ -1514,6 +1538,31 @@ export default class API {
             .then( resp => resp.data );
     }
 
+    overpassSyncCheck( projectTaskInfo ) {
+        const params = {
+            path: `/grail/overpasssynccheck?projectTaskInfo=${ projectTaskInfo }`,
+            method: 'GET'
+        };
+
+        return this.request( params )
+            .then( resp => this.statusInterval( resp.data.jobid ) )
+            .then( resp => {
+                return {
+                    data: resp.data,
+                    message: 'Overpass sync complete.',
+                    status: 200,
+                    type: 'success'
+                };
+            } )
+            .catch( err => {
+                return {
+                    message: err.data,
+                    status: err.status,
+                    type: err.type
+                };
+            } );
```

Comment:
Do we need the message format if we don't alert the user?

Reply:
We do alert user of this one